src/components/basic/DlPanelContainer/DlPanelContainer.vue

```diff
@@ -282,13 +301,12 @@ export default defineComponent({
                 textAlign: 'left',
                 fontSize: 'var(--dl-font-size-body)',
                 position: 'relative',
-                padding: '0px 10px',

```

Comment:
Removing this is affecting other components, is it on purpose ?

Reply:
This might affect other places, but without removing this side panel changes are not possible